sql2parquet/sql2parquet.py

```diff
@@ -0,0 +1,120 @@
+from botocore.exceptions import ClientError
+from pathlib import Path, PureWindowsPath, PurePosixPath
+from sqlalchemy import create_engine
+from time import time
+from typing import Dict
+import pandas as pd
+import boto3, glob, logging, os, sys, toml
+
+CONFIG_FILE_NAME = 'config.toml'
+OUTPUT_DIRECTORY = 'parquet'
+FILE_EXT = '.parquet'
+AWS_S3_BUCKET = 'ensembl-genome-data-parquet'
+
+logging.basicConfig(filename="log.txt", level=logging.DEBUG, format='%(asctime)s %(levelname)s %(module)s %(name)s %(message)s')
+logger = logging.getLogger(__name__)
+
+
+def read_config(config_file_name: str) -> Dict[str, str]:
+    '''
+    Read TOML file containing the DB info
+
+    Args:
+        config_file: TOML configuration file name
+
+    Returns:
+        Dictionary containing the parsed data
+
+    Raises:
+        FileNotFoundError: If `config_file` doesn't exist
+        Exception: Any exception encountered except FileNotFoundError during reading TOML config.
+    '''
+
+    try:
+        return toml.load(open(config_file_name))
+    except FileNotFoundError:
+        logger.error('%s file doesn\'t exist!', config_file_name)
+        sys.exit('%s file doesn\'t exist!', config_file_name)
+    except Exception as e:
+        logger.error(e)
+        sys.exit(e)
+
+def sqlToParquet(config: Dict[str, str]) -> None:
+    '''
+    Read tables from remote SQL server and exports the data as parquet files to OUTPUT_DIRECTORY
+
+    Args:
+        config: Dictionary config object
+    '''
+
+    if not os.path.exists(OUTPUT_DIRECTORY): os.mkdir(OUTPUT_DIRECTORY)
+
+    for database in config['databases']:
+        for species in database['species']:
+
+            db_connection_str = f'mysql+pymysql://{database["db_user"]}@{database["location"]}:{database["port"]}/{species["db_name"]}'
+            engine = create_engine(db_connection_str)
+
+            for table in database['tables']:
+                logger.info('Exporting %s table for %s...', table["table_name"], species["species_name"])
+
+                df = pd.read_sql(table['query'], con=engine, params={'species_name': species['species_name']})
+
+                if not os.path.exists(os.path.join('parquet', f'data={table["table_name"]}')): os.mkdir(os.path.join('parquet', f'data={table["table_name"]}'))
```

Comment:
`'parquet'` param can be replaced with `OUTPUT_DIRECTORY`
```
if not os.path.exists(os.path.join(OUTPUT_DIRECTORY, f'data={table["table_name"]}')): os.mkdir(os.path.join(OUTPUT_DIRECTORY, f'data={table["table_name"]}'))
```
Same for the line below (line 65)

Reply:
Thank you for pointing it out, don't know how I missed it, must have messed it up while refactoring
Fixed it in latest commit